elements/lisk-validator/test/lisk_validator_keywords.spec.ts

```diff
@@ -85,6 +85,320 @@ describe('validator keywords', () => {
 				]),
 			);
 		});
+
+		describe('dataType value validation', () => {
+			describe('string', () => {
+				it('should be return empty error if valid', () => {
+					const result = validator.validate(
+						{
+							...validSchema,
+							properties: { myProp: { dataType: 'string', fieldNumber: 1 } },
+						},
+						{ myProp: 'string' },
+					);
+					expect(result).toBeEmpty();
+				});
+
+				it('should be invalid if minLength is not satisfied', () => {
+					const result = validator.validate(
+						{
+							...validSchema,
+							properties: {
+								myProp: { dataType: 'string', fieldNumber: 1, minLength: 3 },
+							},
+						},
+						{ myProp: 'ch' },
+					);
+					expect(result).toHaveLength(1);
+				});
+
+				it('should be invalid if maxLength is not satisfied', () => {
+					const result = validator.validate(
+						{
+							...validSchema,
+							properties: {
+								myProp: { dataType: 'string', fieldNumber: 1, maxLength: 3 },
+							},
+						},
+						{ myProp: 'change' },
+					);
+					expect(result).toHaveLength(1);
+				});
+
+				it('should be invalid if not string', () => {
+					const result = validator.validate(
+						{
+							...validSchema,
+							properties: {
+								myProp: { dataType: 'string', fieldNumber: 1, maxLength: 3 },
+							},
+						},
+						{ myProp: 32 },
+					);
+					expect(result).toHaveLength(1);
+				});
+			});
+
+			describe('bytes', () => {
+				it('should be return empty error if valid', () => {
```

Comment:
"should ~~be~~ return an empty error if valid" and also change it in the first test of each test suite